src/test/java/teammates/test/cases/common/FieldValidatorTest.java

```diff
@@ -665,7 +660,7 @@ public void test_REGEX_GOOGLE_ID_NON_EMAIL() throws Exception {
     private void runGenericTestCasesForCappedSizeStringTypeField(
             FieldType fieldType, 
             int maxSize, 
-            String errorMessageFormat, 
+            String errorMessage,
```

Comment:
Again, because invalid input and specific reason are no longer displayed, string formatting is not needed, so the errorMessage can be passed in directly.